### DISCUSSION TITLE
Full Version?

### DISCUSSION THREAD

**Discussion by taunnt** [TeamAmaze/AmazeFileUtilities-Issue-Tracker#78] [Q&A]
Hello I installed this on a Quest 2. Issue I keep seeing is when it tries to download a add-on for video it fails. I see a black screen and three dots for over 10 minutes. Is there a full version from Github I can download? Or just the add-on?

**Comment by EmmanuelMess** [comment]
> Hello I installed this on a Quest 2.

I had no idea you could do that.

> Issue I keep seeing is when it tries to download a add-on for video it fails.

Hmm maybe an issue with links, @VishalNehra ?


> Is there a full version from Github I can download? Or just the add-on?

I think you can try the demo. But I don't know the architecture, try [this](https://github.com/TeamAmaze/AmazeFileManager/files/9174223/amaze-utils-arm64-v8a-v0.1-304.zip) then [this](https://github.com/TeamAmaze/AmazeFileManager/files/9174225/amaze-utils-armeabi-v7a-v0.1-304.zip).
